cognite/client/utils/_priority_tpe.py

```diff
@@ -0,0 +1,131 @@
+# type: ignore
+"""
+This code has been modified from the original created by Oleg Lupats, 2019, under an MIT license:
+project = 'PriorityThreadPoolExecutor'
+url = 'https://github.com/oleglpts/PriorityThreadPoolExecutor'
+copyright = '2019, Oleg Lupats'
+author = 'Oleg Lupats'
+release = '0.0.1'
+
+MIT License
+
+Copyright (c) 2019 The Python Packaging Authority
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+"""
+
+import atexit
```

Comment:
This file is missing type annotations. Did you reenable mypy?

Reply:
Check out line 0 😅 I was hoping for some leniency here - or perhaps a shared effort 😬 Pretty much the same code as in the CPython source: https://github.com/python/cpython/blob/3.8/Lib/concurrent/futures/thread.py (except for the different queue ++)

Reply:
# 😇

Reply:
Leniency granted, but if ignoring entire files, please add them to the mypy.ini file instead of adding a catch all `# type: ignore`. Easier to get an overview over what is missing type annotations then, so we can address it later.

Reply:
Thanks! 😄 Ah, yes, that sounds way better 👌